docs/architecture/grpc-for-wcf-developers/why-grpc.md

```diff
@@ -1,11 +1,41 @@
 ---
 title: Why gRPC is recommended for WCF developers - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: A discussion of why gRPC is a good fit for WCF developers looking to migrate to modern architectures and platforms.
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # Why gRPC is recommended for WCF developers
 
+Before diving deeply into the language and techniques of gRPC, it's worth pulling together some ideas about why gRPC is the right solution for WCF developers who want to migrate to .NET Core, given there are alternatives available.
```

Comment:
worth pulling together some ideas about > maybe saying "it's worth discussing" would be better here? more direct?